nixos/modules/tasks/filesystems/zfs.nix

```
@@ -216,6 +218,23 @@ in
   ###### interface
 
   options = {
+    boot.initrd.zfs = {
+      extraPools = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        example = [ "encrypted_data" ];
+        description = ''
+          Name or GUID of extra ZFS pools that you wish to import during boot stage 1.
+
+          Most setups should use {option}`boot.zfs.extraPools` to import extra pools during
+          stage 2 instead. The root pool(s) do not need to be listed here, as they are
+          automatically imported based on NixOS's {option}`fileSystems` options.
+
+          This option is needed when encrypted pools will be unlocked over SSH as the
+          SSH server in {option}`boot.initrd.network.ssh` only runs during stage 1.
```

I think this could be worded a bit better. Maybe  something like this?
```suggestion
          This option is needed when encrypted pools will be unlocked over SSH as the
          SSH server during stage 2 does not start until after file systems are mounted.
```